projects/demo/src/modules/experimental/icon/icon.style.less

```diff
@@ -1,5 +0,0 @@
-@supports not (mask-mode: luminance) {
-    .luminance {
-        display: none;
```

Comment:
Delete this line too:

https://github.com/taiga-family/taiga-ui/blob/425be2617236e1b7cfd5107824e7e7e270515ece/projects/demo-playwright/tests/demo/is-flaky-examples.ts#L16